dashboard2/cookbook/headers.md

```diff
@@ -13,7 +13,7 @@ Request headers contain additional information on the client viewing the site an
 + `X-TranslationProxy-Translating-Host: jp.eveonline.com`: This header contains the domain under which the Proxy is serving the translated site.
 + `X-TranslationProxy-Originator-IP: 192.168.168.192`: If enabled, this header contains the IP address of the requester, which may be hidden by CDNs and other proxies.
 + Future headers of the `X-TranslationProxy` containing other metadata
-+ `User-Agent`: This header is somewhat special, in the sense that it's not specific to The Proxy, rather, it is sent by almost all browsers to identify themselves. The reason it finds a place in this article is that it can be used to identify Proxy-served requests. Google AppEngine modifies this header when sending requests, in a way that ensure no further tampering before the header reaches the original server, by adding `AppEngine-Google; (+http://code.google.com/appengine; appid: u~skawa-easyling)` - this can be used to whitelist the Proxy.
++ `User-Agent`: This header is somewhat special, in the sense that it's not specific to The Proxy, rather, it is sent by almost all browsers to identify themselves. The reason it finds a place in this article is that it can be used to identify Proxy-served requests. Google AppEngine modifies this header when sending requests, in a way that ensure no further tampering before the header reaches the original server, by adding `AppEngine-Google; (+http://code.google.com/appengine; appid: u~{{whitelabel-install}})` - this can be used to whitelist the Proxy.
```

Comment:
But this is not true, is it? In practice, it will (almost) always say `skawa-easyling`.

Reply:
Well, as true as I can go without mentioning Easyling. The point of these changes was to remove all references to the brand and make the docs truly whitelabel-friendly. I can't just leave this one in there, can I?

Reply:
How about using this: `AppEngine-Google; (+http://code.google.com/appengine; appid: {{application-name}})`?

If we're being technically correct, the `u~` at the beginning is also subject to change, since that's the region ID, and Easyling deployed in a different region will receive a different region letter.

Reply:
Sure, sounds good. I didn't realise what the `u~` was.